doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/index.md

```diff
@@ -29,7 +29,7 @@ To build a C++ SDK, you need to first install the following third-party dependen
 * SQLite3 (optional)
 
 The installation process may vary depending on the target platform.
-For platform-specific instructions, see the corresponding subsections.
+For detailed platform-specific instructions, refer to [Supported platforms]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/Supported-platforms/).
```

Comment:
This page no longer exists: it's merged into Programming-guide/Using-Kaa-endpoint-SDKs/. Please update the link.